packages/vanilla/src/sass/components/banner/_mixin.scss

```diff
@@ -1,53 +1,42 @@
-//@use '../../base/colors' as *;
-// @use '../button' as *;
+@use '../../layout/container' as *;
 
 @mixin banner-info($bg-color) {
   background-color: $bg-color;
-  padding-top: 16px;
-  padding-bottom: 16px;
+  padding-top: var(--cbp-space-4x);
+  padding-bottom: var(--cbp-space-4x);
 
   & > div {
     @content;
-    margin: 0 16px;
+    @extend .cbp-fluid-container;
 
     & > div:first-child {
       display: flex;
       flex-shrink: 0;
     }
-
-    @media (min-width: 600px) and (max-width: 1439px) {
-      margin: 0 20px;
-    }
-
-    @media (min-width: 1440px) {
-      margin: 0 28px;
-    }
   }
 }
 
 @mixin banner-text($text-color) {
   color: $text-color;
   font-size: 14px;
   font-weight: 400;
-  line-height: 1.5;
-  margin-top: 16px;
-  margin-bottom: 16px;
-  max-width: 1024px;
+  line-height: var(--cbp-line-height-sm);
+  margin-top: var(--cbp-space-4x);
+  margin-bottom: var(--cbp-space-4x);
+  max-width: 100%; 
 }
 
 @mixin banner-title($text-color) {
   color: $text-color;
-  font-size: 20px;
-  font-weight: 500;
-  line-height: 1.5;
+  font-size: var(--cbp-font-size-masthead-2);
```

Comment:
Same as the previous `font-size` comment for the Banner titles.

Reply:
The other was actually the icon size. Updated this one to heading-md, which is 20px per design spec.